lib/semantics/resolve-names.cc

```diff
@@ -2237,7 +2199,14 @@ void InterfaceVisitor::CheckGenericProcedures(Symbol &generic) {
   ResolveSpecificsInGeneric(generic);
   auto &details{generic.get<GenericDetails>()};
   if (auto *proc{details.CheckSpecific()}) {
-    SayAlreadyDeclared(generic.name(), *proc);
+    auto msg{
```

Comment:
Is this check deferred until it is certain that all of the specific procedures for the generic interface have been seen, including the cases where multiple `GENERIC` statements &/or generic interface blocks with the same generic name are used?

Reply:
Yes, this is called from `FinishSpecificationPart` at the end of the specification part, so they should all have been seen by then.